darts/models/forecasting/ensemble_model.py

```diff
@@ -377,6 +386,117 @@ def _predictions_reduction(
             ]
         return predictions[0] if is_single_series else predictions
 
+    def save(
+        self, path: Optional[Union[str, os.PathLike, BinaryIO]] = None, **pkl_kwargs
+    ) -> None:
+        """
+        Saves the ensemble model under a given path or file handle.
+
+        If `forecasting_models` contains TorchForecastingModel, two files (model object and checkpoint) would be saved.
+
+        Example for saving and loading a :class:`RegressionEnsembleModel`:
+
+            .. highlight:: python
+            .. code-block:: python
+
+                from darts.models import RegressionEnsembleModel, LinearRegressionModel, TiDEModel
+
+                model = RegressionEnsembleModel(
+                    forecasting_models = [
+                        LinearRegressionModel(lags=4),
+                        TiDEModel(input_chunk_length=4, output_chunk_length=4),
+                        ],
+                        regression_train_n_points=10,
+                )
+
+                model.save("my_ensemble_model.pkl")
+                model_loaded = RegressionEnsembleModel.load("my_ensemble_model.pkl")
+            ..
+
+        Parameters
+        ----------
+        path
+            Path or file handle under which to save the ensemble model at its current state. If no path is specified,
+            the ensemble model is automatically saved under ``"{RegressionEnsembleModel}_{YYYY-mm-dd_HH_MM_SS}.pkl"``.
+            If the ith model of `forecasting_models` is a TorchForecastingModel, two files (model object and checkpoint)
+            would be saved under ``"{path}.{ithModelClass}_{i}.pt"`` and ``"{path}.{ithModelClass}_{i}.ckpt"``
+            E.g., ``"RegressionEnsembleModel_2024-07-25_14_58_53.pkl"``
+            ``"RegressionEnsembleModel_2024-07-25_14_58_53.pkl.TiDEModel_1.pt"``
+            ``"RegressionEnsembleModel_2024-07-25_14_58_53.pkl.TiDEModel_1.ckpt"``
+        pkl_kwargs
+            Keyword arguments passed to `pickle.dump()`
+        """
+
+        if path is None:
+            # default path
+            path = self._default_save_path() + ".pkl"
+
+        if isinstance(path, (str, os.PathLike)):
+            # save the whole object using pickle
+            with open(path, "wb") as handle:
+                pickle.dump(obj=self, file=handle, **pkl_kwargs)
+        elif isinstance(path, io.BufferedWriter):
+            # save the whole object using pickle
+            pickle.dump(obj=self, file=path, **pkl_kwargs)
+        else:
+            raise_log(
+                ValueError(
+                    "Argument 'path' has to be either 'str' or 'PathLike' (for a filepath) "
+                    f"or 'BufferedWriter' (for an already opened file), but was '{path.__class__}'."
+                ),
+                logger=logger,
+            )
```

Comment:
we can use the existing save logic:

```suggestion
        super().save(path, **pkl_kwargs)
```